usr/lib/common/h_extern.h

```diff
@@ -2478,6 +2478,17 @@ CK_RV ecdsa_priv_unwrap_get_data(TEMPLATE *tmpl, CK_BYTE *data,
 CK_RV ec_priv_unwrap(TEMPLATE *tmpl,
                      CK_BYTE *data, CK_ULONG data_len, CK_BBOOL isOpaque);
 
+// Dilithium routines
+//
+CK_RV dilithium_publ_check_required_attributes(TEMPLATE *tmpl, CK_ULONG mode);
```

Comment:
Its a little weird to have the headers in a separate commit from the function bodies. Id rather squash the first to commits (dilithium support) and keep the test cases in an extra commit.

Reply:
.. or at least just move the headers to the second commit if you want to keep the define stuff separate.

Reply:
No that's fine. I almost decided to put the first two commits into one myself.

Reply:
Just squashed the first two commits into one.